examples/annotated_diffusion.ipynb

```diff
@@ -0,0 +1,1602 @@
+{
```

Comment:
<p>Doesn't this cell error when running the notebook top to bottom?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>